ci/documentation/preview.sh

```diff
@@ -22,6 +22,7 @@ function openBrowser() {
 
 openBrowser
 
+# NB: Can't use strict mode with serving. Use lint.sh for checking.
```

Comment:
Strict mode is used for publishing to production. It helps to find invalid local links, menus, and so on.
Don't know a way to enable it in serving